test/models/test_step.py

```diff
@@ -73,49 +82,54 @@ def test_wrong_material_matchrules(self):
     self.assertFalse(self.step.validate())
```

Comment:
I wonder why the test does not fail here. Shouldn't validate always return `None`?